pkg/driver/mocks/mock_mount.go

```diff
@@ -203,3 +206,17 @@ func (mr *MockMounterMockRecorder) Unmount(arg0 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Unmount", reflect.TypeOf((*MockMounter)(nil).Unmount), arg0)
 }
+
+// canSafelySkipMountPointCheck mocks base method.
+func (m *MockMounter) canSafelySkipMountPointCheck() bool {
```

Comment:
I'm confused on the purpose of this method, do you mind clarifying?

Reply:
Package mocks is a generated GoMock package. This was auto-generated using the hack/update-gomock. Not sure what the specific use-case is here but I think we should leave it